### PR TITLE
Update makefile scaffold

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by fredj** [comment]
LGTM, maybe we should create an (empty ?) `static-ngeo/js/components/` directory in the scaffold

File: c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Directives and their partials are structured by components
+
+A subdirectory is created for each component.
+The component partials are directly stored in the subdirectory.
+
+The directive template URLs must be of the form "templateUrl: app.baseTemplateUrl + subdirectory + '/' + 'partial\_name.html'".
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md]: We steal have a partial folder?

File: c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/+package+module.js_tmpl [hunk 2]
```diff
@@ -18,4 +18,4 @@ goog.require('gmf');
  * The default template based URL, used as it by the template cache.
  * @type {string}
  */
-{{package}}.baseTemplateUrl = '{{package}}';
+{{package}}.baseTemplateUrl = '{{package}}/';
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/+package+module.js_tmpl]: Why having an end '/' as I remember we don't have it on ngeo...

File: c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md]: not better:
`app.baseTemplateUrl + '/<component>/<template>.html'`
?

Comment (gberaudo) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md]: No more partial folder. Updated the doc with your suggestion.

File: c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/+package+module.js_tmpl (hunk 2 above)
Comment (gberaudo) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/+package+module.js_tmpl]: Indeed, fixed.

File: c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+A subdirectory is created for each component.
+The component partials are directly stored in the subdirectory.
+
+The partials are loaded individually at runtime in debug mode but preloaded in a template cache for production.
+For this mechanism to work correctly the directive template URL must follow some conventions:
+
+In the directive, the template url is written as follow: "templateUrl: {{package}}.baseTemplateUrl + '/<component>/<partial>.html'".
+
+In the main html file, the debug section should contain "{{package}}.templateBaseUrl = '${request.static_url("{{package}}:static-ngeo")}';"
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl]: Shouldn't be also added here?
https://github.com/camptocamp/c2cgeoportal/blob/aa4568be45b4c8694b70610cb71b7ce8b4c1d554/c2cgeoportal/scripts/import_ngeo_apps.py#L229

**Comment by sbrunner** [comment]
Looks good for me :-)

**Comment by gberaudo** [comment]
Thanks for the review.

**Pull request merged**